src/CliCommand.php

```diff
@@ -97,10 +97,19 @@ public function __invoke(array $args, array $options) {
       $dump = new IMysqldump('mysql:host=' . DB_HOST . ';dbname=' . DB_NAME, DB_USER, DB_PASSWORD, [
         'add-drop-table' => TRUE,
       ]);
-      $tableWheres = apply_filters(static::PREFIX . '/table-wheres', [
-        "{$wpdb->prefix}comments" => "comment_post_ID IN ({$allowedOrderIds})",
-        "{$wpdb->prefix}users" => "ID IN ({$allowedUserIds})",
-        "{$wpdb->prefix}usermeta" => "user_id IN ({$allowedUserIds})",
+
+      if ($allowedUserIds) {
+        $tableWheres["{$wpdb->prefix}users"] = "ID IN ({$allowedUserIds})";
+        $tableWheres["{$wpdb->prefix}usermeta"] = "user_id IN ({$allowedUserIds})";
+      }
+      if ($allowedOrderIds) {
+        $tableWheres["{$wpdb->prefix}comments"] = "comment_post_ID IN ({$allowedOrderIds})";
+        $tableWheres["{$wpdb->prefix}woocommerce_order_items"] = "order_id IN ({$allowedOrderIds})";
+      }
+      if ($allowedOrderItemIds) {
+        $tableWheres["{$wpdb->prefix}woocommerce_order_itemmeta"] = "order_item_id IN ({$allowedOrderItemIds})";
+      }
```

Comment:
Can we move these in the WooCommerce group/block below?

Reply:
Not sure what you mean here, but we can refactor later on.